org.flatpak.rmac.yml

```diff
@@ -0,0 +1,23 @@
+id: org.flatpak.rmac
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: rmac
+modules:
+  - name: rmac
+    buildsystem: simple
+    build-commands:
+      - install -D rmac /app/bin/rmac
```

Comment:
this is no appstream file installed, nor .desktop or icon.

Reply:
It's a cli program, is an icon absolutely necessary?

Reply:
appstream file still missing, desktop file never installed.